server/main.lua

```diff
@@ -0,0 +1,143 @@
+---@class CreateEntityQuery
+---@field license string The license of the owner
```

Comment:
Maybe we should take out license here? We could lookup the license of the player given the citizenid to make the API simpler

Reply:
didn't remove the field